docs/Documentation/Machine_Learning/index.md

```diff
@@ -0,0 +1,131 @@
+```
+conda activate /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/pt
+
+conda install pytorch torchvision torchaudio cpuonly -c pytorch
+```
+Answer yes to proceed, and you should be up and running with PyTorch! The [PyTorch](https://pytorch.org/) webpage has great resources for getting started, including resources on [learning the basics](https://pytorch.org/tutorials/beginner/basics/intro.html) and [PyTorch recipies](https://pytorch.org/tutorials/recipes/recipes_index.html).
+
+###Getting started with TensorFlow
+
+Getting started with TensorFlow is similar to process for PyTorch. The first step is to construct anempty conda environment to work in:
+```
+module load conda
+
+conda create -p /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/tf python=3.9
+```
+Subsequently, activate the tf conda environment, ensure you are running the latest version of pip in your environment, and install the CPU only version of TensorFlow using pip:
+```
+conda activate /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/tf
+pip install --upgrade pip
+pip install tensorflow-cpu
+```
+You should now be up and running with a TensorFlow! Similar to PyTorch, the [TensorFlow webpage](https://www.tensorflow.org/learn) has lots of great resources for getting started, including turotials, basic examples, and more! 
+
+Include a section on how to run the job, e.g., with job script examples or commands for an interactive session.
+
+### Example Job Script
+
+??? example "PyTorch or TensorFlow CPU"
+	```
+	#!/bin/bash 
+	#SBATCH --nodes=1			# Run the tasks on the same node
+	#SBATCH --time=1:00:00			# Required, estimate 1 hour
+	#SBATCH --account=<your_account>
+	#SBATCH --partition=debug
+	#SBATCH --exclusive			# if you want to use the whole node
+
+	module load conda 
+
+	cd /projects/<your_project_here>/<your_code_directory>
+
+	conda activate /projects/<your_project_here>/<folder_for_conda_envs>/pt #or tf
+
+	srun python your_pt_code.py
+
+	```
+!!! note
+	This Getting Started section is only scratching the surface of ML libraries and resources that can be used on Kestrel. Tools such as LightGBM, XGBoost, and scikit-learn work well with conda environments, and other tools such as Flux for the Julia Language can be used on Kestrel as well.
+
+Once you have completed your batch file, submit using
+```
+sbatch <your_batch_file_name>.sb
+```
+
+## Supported Versions
+
+| Kestrel | Swift | Vermillion |
+|:-------:|:-----:|:----------:|
+| 0.0.0   | 0.0.0 | 0.0.0      |
+
+## Advanced
+
+The above examples are simple CPU-based computing environments. To build conda environments for GPUs we refer you to the [PyTorch](https://pytorch.org/get-started/locally/) and [TensorFlow](https://www.tensorflow.org/install) install directions. 
+
+For optimized TensorFlow performance, we recommend using a [containerized version of TensorFlow](https://). 
```

Comment:
Missing link

Reply:
Provided path to Containerized_TensorFlow/index.md